src/main/java/net/minecraftforge/fml/common/registry/EntityEntry.java

```diff
@@ -73,7 +67,7 @@ public Entity newInstance(World world)
         }
         catch (Exception e)
         {
-            FMLLog.log.error("Error creating entity.", e);
+            FMLLog.log.error("Encountered an exception while constructing entity '" + this.getRegistryName() + '\'', e);
```

Comment:
Use `{}` for string formatting here

Reply:
Nope. There's no `error` method that accepts arguments *and* a `Throwable`. It will consider the `Throwable` a formatting argument.

Reply:
It won't, if it's an extra argument at the end (i.e. there are more arguments passed than used in the log message).